recipes/go/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Move the go directory into $PREFIX/ so it can be built in its install location.
+cd .. && mv go $PREFIX/ && cd $PREFIX/go
+
+# Change to `src` build and then move out.
+cd src
+./all.bash
+cd ..
+
+# Delete `test` directory as it is no longer needed.
+rm -rf test
```

Comment:
We should also chat about where we want this to live. We could put it in `$PREFIX/share` or similar so that it doesn't look so weird.